alpaca/rest_test.go

```diff
@@ -941,7 +1044,7 @@ func TestGetOptionContracts(t *testing.T) {
 		UnderlyingSymbols:     "some_symbol",
 		ShowDeliverable:       true,
 		Status:                OptionStatusActive,
-		ExpirationDate:        civil.Date{Year: 2000, Month: 01, Day: 01},
+		ExpirationDate:        civil.Date{Year: 2000, Month: 0o1, Day: 0o1},
```

Comment:
[minor] Should be 1s

Reply:
Apologies, vscode changed this automatically when I saved the file.
3f42f59d613707cd881b558d240584c917ca1566